docs/monoDrive_home/Simulator-Configuration.md

```diff
@@ -2,135 +2,132 @@
-      "Road":{
-         "specular_exponent":1.0,
-         "specular_coefficient":0.1,
-         "diffuse_coefficient":0.65,
-         "dielectric_constant":7.0,
-         "roughness":0.65
+      "Plastic": {
+         "dielectric_constant": 2.8,
+         "diffuse_coefficient": 0.6,
+         "roughness": 0.09,
+         "specular_coefficient": 0.4,
+         "specular_exponent": 25.0
       },
-      "RoadPaint":{
-         "specular_exponent":10.0,
-         "specular_coefficient":10.0,
-         "diffuse_coefficient":2.0,
-         "dielectric_constant":7.0,
-         "roughness":1.0
+      "Road": {
+         "dielectric_constant": 7.0,
+         "diffuse_coefficient": 0.65,
+         "roughness": 0.65,
+         "specular_coefficient": 0.1,
+         "specular_exponent": 1.0
       },
-      "Steel":{
-         "specular_exponent":26.0,
-         "specular_coefficient":1.0,
-         "diffuse_coefficient":0.33,
-         "dielectric_constant":0.0,
-         "roughness":0.23
+      "Steel": {
+         "dielectric_constant": 0.0,
+         "diffuse_coefficient": 0.33,
+         "roughness": 0.23,
+         "specular_coefficient": 1.0,
+         "specular_exponent": 26.0
       },
-      "Tire":{
-         "specular_exponent":25.0,
-         "specular_coefficient":0.40,
-         "diffuse_coefficient":0.60,
-         "dielectric_constant":2.8,
-         "roughness":0.15
+      "Tire": {
+         "dielectric_constant": 2.8,
+         "diffuse_coefficient": 0.6,
+         "roughness": 0.15,
+         "specular_coefficient": 0.4,
+         "specular_exponent": 25.0
       },
-      "Tree":{
-         "specular_exponent":10.0,
-         "specular_coefficient":0.05,
-         "diffuse_coefficient":0.97,
-         "dielectric_constant":10.0,
-         "roughness":0.60
+      "Tree": {
+         "dielectric_constant": 10.0,
+         "diffuse_coefficient": 0.97,
+         "roughness": 0.6,
+         "specular_coefficient": 0.05,
+         "specular_exponent": 10.0
       },
-      "Wheel":{
-         "specular_exponent":92.0,
-         "specular_coefficient":0.95,
-         "diffuse_coefficient":0.06,
-         "dielectric_constant":10.0,
-         "roughness":0.15
+      "Wheel": {
+         "dielectric_constant": 10.0,
+         "diffuse_coefficient": 0.06,
+         "roughness": 0.15,
+         "specular_coefficient": 0.95,
+         "specular_exponent": 92.0
       }
+   },
+   "server_ip": "127.0.0.1",
+   "server_port": 8999,
+   "simulation_mode": 1,
+   "traffic_configuration": {
+      "max_vehicles": 10,
+      "spawn_rate": 0.25
    }
 }
 ```
 <p>&nbsp;</p>
 
 ### Configuration Tags
 
-- **server_ip**: The ip address of the computer running the simulator. If both simulator and client are running on the same computer use localhost as the ip address (127.0.0.1).
+- **client_ip**: The ip address of the computer running the simulator. If both simulator and client are running on the same computer use localhost as the ip address (127.0.0.1).
```

Comment:
This is actually an addition to the configuration. For some reason the `server_ip` was dropped in the config file I sent to you. Can you add `server_ip` back into the mix and use this description for client and server:

```
**client_ip**: The IP address of the computer running the client software. 
**server_ip**: The IP address of the computer running the monoDrive simulator.
```